stdlib/int64.gr

```diff
@@ -281,18 +520,37 @@ export let rec gte = (x: Int64, y: Int64) => {
   ret
 }
 
-// Int64 bit/logical operations
+/**
+ * @section Logical: Int64 Logical Functions.
+ */
+
+/**
+ * Computes the bitwise logical "not" of the given value.
+ *
+ * @param value: The given value
+ * @returns The bitwise logical "not" of the given value.
```

Comment:
```suggestion
 * @returns The bitwise logical "not" of the given value
```
This needs better wordsmitting. I really don't like when the "returns" is the same as the description (it's really lazy and uninformative). cc @ospencer

Reply:
we could say plus one, but also oscar makes a good point we could probably just remove the returns for this one because of how self explanatory it is.

Reply:
```suggestion
 * @returns The complement of the given value
```
Wikipedia says this is "the complement"

Otherwise, we could do something like (similar to MDN):
```suggestion
 * @returns Containing the inverted bits of the given value
```

Reply:
I think the mdn one would work `complement` gets the point across but i also think it is kind of above normal wording so beginners probably won't understand what it is. up to you guys though.

Reply:
I like the inverted bits version!